resource_management/azure_mgmt_compute/spec/virtual_machines_spec.rb

```diff
@@ -148,20 +150,12 @@
     # to find out whether VM is turned on or not. So the timeout is added here.
     # todo: add VM polling until VM is started if API method appear which provides
     # such info.
-    sleep 300
+    # sleep 300
```

Comment:
Yup that's better. Updated it based on one of the env variable that depends on recording vs playback.